planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -3300,12 +3309,15 @@ void AvoidanceModule::insertWaitPoint(
     return;
   }
 
-  const auto stop_distance = helper_.getMildDecelDistance(0.0);
-  if (stop_distance) {
-    const auto insert_distance = std::max(data.to_stop_line, *stop_distance);
+  if (data.stop_target_object.get().is_stoppable) {
```

Comment:
// If target object can be stopped for, insert a deceleration point and return

Reply:
Add comment line in https://github.com/autowarefoundation/autoware.universe/pull/3978/commits/0d099f4f4baaa6ff9134f77160612075a7b5c162.